### PR TITLE
Swoosh

### PR DESCRIPTION
**Pull request by shreyaag770**
I changed dark colors in Swoosh animation with lighter shades to increase the visibility of text.

## Before:

s1

## After:
s3

### CODE REVIEW

**Comment by shreyaag770** [comment]
@zregvart I get your point and tried for a better combination of colors. Please have a look at it.
Also, I remove the changes made on the blog previous/next links.

c1

File: antora-ui-camel/src/css/blog.css [hunk 1]
```diff
@@ -67,7 +67,7 @@ article.blog p {
 }
 
 .blog a.arrow {
-  font-size: 1rem;
+  font-size: 2rem;
```
Comment (zregvart) [inline antora-ui-camel/src/css/blog.css]: This change also belongs only to #193

**Comment by shreyaag770** [comment]
Done! @zregvart and I am really sorry, I forgot one change earlier.

**Comment by shreyaag770** [comment]
oh! I didn't know about squashing commits. Thanks a lot! @zregvart but I guess, I have lost this branch or I don't know, some error is there can I send another PR?

**Comment by shreyaag770** [comment]
Oh! I didn't know about squashing commits. Thanks a lot! @zregvart but I guess, I have lost this branch or I don't know some error is there. I have sent another PR #263 for this. Please review #263 .

File: antora-ui-camel/src/css/blog.css [hunk 2]
```diff
@@ -153,4 +153,4 @@ article.blog p {
   .blog.list {
     flex-direction: row;
   }
-}
+}
```
Comment (zregvart) [inline antora-ui-camel/src/css/blog.css]: This change is causing lint check to fail.

File: antora-ui-camel/src/css/frontpage.css [hunk 3]
```diff
@@ -249,7 +249,11 @@ section.frontpage h1 {
...
 
+  section.frontpage.projects .project p:nth-child(2) {
+    text-align: center;
+  }
+
   section.frontpage.projects .project p:nth-child(3) {
     text-align: center;
   }
-}
+}
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css]: We need a newline at the end of the file

**Comment by zregvart** [comment]
Closing in favour of #263

**Pull request closed without merging**